src/torchcodec/_core/custom_ops.cpp

```diff
@@ -308,6 +318,7 @@ void add_video_stream(
     std::optional<std::string_view> dimension_order = std::nullopt,
     std::optional<int64_t> stream_index = std::nullopt,
     std::optional<std::string_view> device = std::nullopt,
+    std::optional<std::string_view> device_variant = std::nullopt,
```

Comment:
Elsewhere, the `device_variant` has a default value of `"default"`. But here, it's defaulted to `std::nullopt`. The latter is consistent with the rest of the parameters to `add_video_stream()`, but it's not self-consistent with all other instances of `device_variant`. It might be better to just make it a non-optional parameter with a default value of `"default"`, and same with `device`.